git.md

```diff
@@ -195,6 +195,26 @@ Fix nasty little bug ISSUE-51 #resolve #time 2h30m
 * Keep the history *clean* and *simple*. Before pushing any branch to the
   public always ensure that it conforms to the style-guide. If it doesn't:
   squash, rebase, reword commits as necessary.
+  
+* When merging multiple small commits into a single change, make sure to
+  keep high locality of the changes. In another words, if you're changing,
```

Comment:
What does "group changes in small vicinity of each other" mean in `git` commit context?

Reply:
It means changes in the same function or same class or same logical unit of the code. The goal here is to disqualify groupings which include changes happening all over the place.

Reply:
Hmm, I would use an real life example here as it is not as clear in the text.